test/test_testing.py

```diff
@@ -813,187 +813,152 @@ def assertRaisesRegexs(self, expected_exception: Type[Exception], *expected_rege
         else:
             raise AssertionError(f"{expected_exception.__name__} was not raised.")
 
-    @onlyCPU
-    def test_complex_support(self, device):
-        actual = torch.ones(1, dtype=torch.float32, device=device)
-        expected = torch.ones(1, dtype=torch.complex64, device=device)
+
+class TestAsserts(TestCase, _TestAssertsMixin):
```

Comment:
Keep this part of the device generic framework but instantiate its tests only for the CPU